card/mod/05_standard/set/all/rich_html/modal.rb

```diff
@@ -43,7 +45,7 @@ def default_modal_content_args args
   end
 
   view :modal_title do |args|
-    _render_title args.merge(:title_class=>'modal-title')
+    "<h4>#{_render_title args.merge(:title_class=>'modal-title')}</h4>"
```

Comment:
What do you mean by "header"? A <header> tag? There is a "modal_header" view with a "modal-header" css class that calls "modal_title"